### PR TITLE
including line streaming visualization

### PR DESCRIPTION
**Pull request by msioufy**
This is a small addition for the api to include line-streaming visualization. Just added a  trait for StreamingVisualization and a appendData method at the visualization class. Note that, in the lineSteaming function, the size parameter is mandatory (not optional). line-streaming doesnt work without the size parameter being specified.

### CODE REVIEW

**Comment by mathisonian** [comment]
Thanks @Siouffy, this is looking good!

It seems like the file `Visualization.scala~` (a save file or something) got checked in on accident. Could you remove that and we'll review and merge.

**Comment by msioufy** [comment]
done ;)

File: src/main/scala/org/viz/lightning/types/Streaming.scala [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package smart.util.lightning.types
```
Comment (freeman-lab) [inline src/main/scala/org/viz/lightning/types/Streaming.scala]: This should be `package org.viz.lightning.types`

File: src/main/scala/org/viz/lightning/Visualization.scala [hunk 2]
```diff
@@ -50,4 +50,12 @@ class Visualization(val lgn: Lightning, val id: Int, val name: String) {
     request.asString.body
   }
 
-}
\ No newline at end of file
+  def appendData(data : Array[Array[Double]]) : Int = {
+    val url = lgn.host + "/sessions/" + lgn.session + "/visualizations/" + this.id + "/data/"
+    implicit val formats = DefaultFormats
```
Comment (freeman-lab) [inline src/main/scala/org/viz/lightning/Visualization.scala:56]: I think you forgot to include the imports here, and also two lines below

**Comment by freeman-lab** [comment]
@Siouffy looks good, thanks a lot for this! if you want to fix these two small things i can merge in.

**Comment by msioufy** [comment]
@mathisonian @freeman-lab  ... Done. I included the lineStreaming function in the "Plots" instead of being in a separate class ... This time I did the testing in isolation of my project so I'm sure I'm not forgetting anything :D ...

**Comment by freeman-lab** [comment]
Excellent, thanks @Siouffy ! Merging in.

**Pull request merged**